pkg/nsx/services/vpc/store_test.go

```diff
@@ -0,0 +1,251 @@
+package vpc
+
+import (
+	"fmt"
+	"reflect"
+	"sync"
+	"testing"
+
+	"github.com/agiledragon/gomonkey"
+	"github.com/stretchr/testify/assert"
+	"github.com/vmware-tanzu/nsx-operator/pkg/config"
+	"github.com/vmware-tanzu/nsx-operator/pkg/nsx"
+	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/ratelimiter"
+	"github.com/vmware-tanzu/nsx-operator/pkg/nsx/services/common"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/bindings"
+	"github.com/vmware/vsphere-automation-sdk-go/runtime/data"
+	"github.com/vmware/vsphere-automation-sdk-go/services/nsxt/model"
+	"k8s.io/client-go/tools/cache"
+)
+
+type fakeQueryClient struct {
+}
+
+func (qIface *fakeQueryClient) List(queryParam string, cursorParam *string, includedFieldsParam *string, pageSizeParam *int64, sortAscendingParam *bool, sortByParam *string) (model.SearchResponse, error) {
```

Comment:
<img width="467" alt="image" src="https://user-images.githubusercontent.com/4385934/203457256-1b44c529-ec86-40aa-85a4-33a517113689.png">
Could you rename those to "_"